### ISSUE TITLE
Difference between GTR_MOT_FPN and GTR_MOTFull_FPN

### ISSUE THREAD

**Issue by pietro-nardelli** [xingyizhou/GTR#35]
Hi, I cannot find in the detail neither here nor in the paper what are the differences between these two models. The configurations are identical but they differ in the training dataset (half vs full).

In the paper you said: "We follow CenterTrack [68] and split each training sequence in half. We use the first half for training and the second half for validation". But the results in table 3 seems obtained by GTR_MOTFull_FPN.

Which of the two models should be considered the "best" one? May I have more information about this? 

Thank you so much in advance.

**Comment by xingyizhou** [comment]
Hi,

  You are all correct. `GTR_MOT_FPN` uses half videos of the training set for training and uses the other half of the training set for validation. `GTR_MOTFull_FPN ` uses full videos of the training set for training and uses all videos of the **test set** for evaluation. Table 3 uses `GTR_MOTFull_FPN` and reports numbers on the test set. Unfortunately, I did not compare which model is better.

Best,
Xingyi
